Jenkinsfile

```diff
@@ -9,13 +9,18 @@ def gitEnv = [
 
 env.CHANGE_BRANCH = env.CHANGE_BRANCH ?: env.BRANCH_NAME
 
+properties([
+    // Run every hour
+    pipelineTriggers([cron('H */1 * * *')])
+])
+
 node('linux && git') {
   stage('checkout') {
     checkout scm
   }
   stage('download updates') {
     sh './update-readmes.sh'
-    sh './update-v4-readmes.sh'
```

Comment:
We probably should keep `sh './update-v4-readmes.sh'` here as it's used to pull in README files. Even after we move docs to the monorepo, we might continue to pull in README files.